util/chplenv/chpl_machine.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import collections
```

Comment:
unused import, so is platform the string one. I'd also prefer to keep stderr and stdout as fully qualified. 

I think you're also missing an import for `error()`